tests/ftests/048-sudo-systemd_create_scope.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: LGPL-2.1-only
+#
+# Create a systemd scope
+#
+# Copyright (c) 2022 Oracle and/or its affiliates.
+# Author: Tom Hromatka <tom.hromatka@oracle.com>
+#
+
+from cgroup import Cgroup as CgroupCli
+from cgroup import CgroupVersion
+from libcgroup import Cgroup
+from systemd import Systemd
+from run import Run
+import ftests
+import consts
+import time
+import sys
+import os
+
+SLICE = 'libcgtests.slice'
+SCOPE = '048delegated.scope'
+pid = None
+
+# Which controller isn't all that important, but it is important that we
+# have a cgroup v2 controller
+CONTROLLER = 'cpu'
+
+def prereqs(config):
```

Comment:
Python expects two empty lines before the start of the function definition.

Reply:
Done